### PR TITLE
Tweak CLI Text!

### PR DESCRIPTION
**Pull request by adamkrellenstein**
Don't merge this before testing it! I may have made a typo somewhere.

### CODE REVIEW

File: tower-cli/towercli/commands/provision.py [hunk 1]
```diff
@@ -140,68 +140,68 @@ def add_args(argparser):
...
         if not args.wlan_ssid:
-            parser_error(message="You must provide a wifi SSID for the router.")
+            parser_error(message="You must provide a WiFi SSID for the router.")
         if not args.wlan_password:
-            parser_error(message="You must provide a wifi password for the router.")
+            parser_error(message="You must provide a WiFi password for the router.")
     else:
         if args.online == args.offline:
-            parser_error(message="You must use one and only one of the argument `--online` and `--offline`.")
+            parser_error(message="You must specify `--online` or `--offline` and not both.")
```
Comment (ouziel-slama) [inline tower-cli/towercli/commands/provision.py:204]: hum.. @adamkrellenstein this message is also displayed when the user does not use either flag (false == false) and not only when he uses both at the same time.

**Comment by ouziel-slama** [comment]
Look great!! just added a comment about an error message!

File: tower-cli/towercli/commands/provision.py (hunk 1 above)
Comment (adamkrellenstein) [reply tower-cli/towercli/commands/provision.py:204]: fixed!

**Comment by adamkrellenstein** [comment]
okay great!

**Pull request merged**